dev/com.ibm.ws.security.common/resources/com/ibm/ws/security/common/internal/resources/SSOCommonMessages.nlsprops

```diff
@@ -32,6 +32,6 @@ CONFIG_REQUIRED_ATTRIBUTE_NULL=CWWKS6104W: The configuration attribute [{0}] tha
 CONFIG_REQUIRED_ATTRIBUTE_NULL.explanation=The specified configuration attribute might be missing from the server configuration. If a value is specified, white space at the beginning and end of the value is removed when the value is processed, so the value in the configuration might be empty or consist of only white space characters.
 CONFIG_REQUIRED_ATTRIBUTE_NULL.useraction=Ensure that the specified configuration attribute is configured, that it is not empty, and that it does not consist of only white space characters.
 
-CONFIG_REQUIRED_ATTRIBUTE_NULL_WITH_CONFIG_ID=CWWKS6105W: The configuration attribute [{0}] that is required is missing or empty in the configuration [{0}] and a default value is not provided. Verify that the attribute is configured, that it is not empty, and that it does not consist of only white space characters.
+CONFIG_REQUIRED_ATTRIBUTE_NULL_WITH_CONFIG_ID=CWWKS6105W: The configuration attribute [{0}] that is required is missing or empty in the configuration [{1}] and a default value is not provided. Verify that the attribute is configured, that it is not empty, and that it does not consist of only white space characters.
 CONFIG_REQUIRED_ATTRIBUTE_NULL_WITH_CONFIG_ID.explanation=The specified configuration attribute might be missing from the specified server configuration element. If a value is specified, white space at the beginning and end of the value is removed when the value is processed, so the value in the configuration might be empty or consist of only white space characters.
```

Comment:
Hi, I just have a couple of wording changes to recommend for this message. It looks good! 

+CONFIG_REQUIRED_ATTRIBUTE_NULL_WITH_CONFIG_ID=CWWKS6105W: The **required** configuration attribute [{0}] is **either** missing or empty in the configuration [{1}] and a default value is not provided. Verify that the attribute is configured, that it is not empty, and that it does not consist of only white space characters.